rs/boundary_node/denylist_updater/src/main.rs

```diff
@@ -474,30 +470,6 @@ impl<T: List> List for WithNormalize<T> {
     }
 }
 
-struct WithRecover<T: List>(T);
-
-#[async_trait]
-impl<T: List> List for WithRecover<T> {
```

Comment:
@DSharifi You deleted it because it is unused?

Reply:
Yes